src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [userName, setUserName] = useState('');
+  const [todoList, setTodoList] = useState(todosFromServer);
+  const [isWarningTitle, setIsWarningTitle] = useState(false);
+  const [isWarningUser, setIsWarningUser] = useState(false);
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+
+    let status = false;
```

Comment:
It would be great to specify what exactly this status it and name it like boolean
![image](https://github.com/mate-academy/react_add-todo-form/assets/96409563/6f51a30b-4218-4920-a039-9c89bf8357b5)